### DISCUSSION TITLE
Issues with echo.js

### DISCUSSION THREAD

**Discussion by boris-glumpler** [HollyIT/filament-static-assets#5] [Q&A] [answered]
Hello,

let me start with that there's a very real possibility that I may be doing things wrong as I'm just starting with Filament. This plugin, however, makes a lot of sense to me and I wish this was built-in, tbh. 

Anyways, I've set up Soketi, installed Laravel Echo and enabled it in `config/filament.php`. Now, a script tag was added to the HTML, but running `filament:cache-assets` doesn't actually cache `echo.js`, so I'm getting a 404 in the console.  Next I tried to add `echo` like so:

```php
Filament::registerScripts([
    'echo' => app(AssetController::class)('echo.js')?->getFile()->getRealPath(),
]);
```

The 404 in the console is gone now and `filament:cache-assets` does cache `echo.js`, but there are now also 2 script tags in the source pointing to the same `echo.js` file, which obviously isn't right neither.

Any help here would be greatly appreciated, cheers!

**Comment by HollyIT** [comment, marked as answer]
Turns out I never added support in for echo.js. I just pushed 0.1.3, so doing a composer update should fix it for you. 

You don't need to do the Filament::registerScripts for echo, as it's already done in the Filament admin package.

**Reply by boris-glumpler** [reply]
Excellent, cheers!
